### PR TITLE
PBM-459 release notes 1.2.0

### PR DESCRIPTION
**Pull request by nastena1606**

### CODE REVIEW

File: doc/source/release-notes/1.2.0.rst [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+*Percona Backup for MongoDB* 1.2.0
+================================================================================
+
+:Date: May 13, 2020
+:Installation: `Installing Percona Backup for MongoDB <https://www.percona.com/doc/percona-backup-mongodb/installation.html>`_
+
+Percona Backup for MongoDB is a distributed, low-impact solution for consistent backups of MongoDB
+sharded clusters and replica sets. This is a tool for creating consistent backups
+across a MongoDB sharded cluster (or a single replica set), and for restoring
+those backups to a specific point in time. The project was
```
Comment (dAdAbird) [inline doc/source/release-notes/1.2.0.rst:13]: > restoring those backups to a specific point in time

won't it be a confusing statement given we do not support PITR yet?

Comment (nastena1606) [reply doc/source/release-notes/1.2.0.rst:13]: I suppose this text was written by marketing.

Comment (akira-kurogane) [reply doc/source/release-notes/1.2.0.rst:13]: We can trim this down. Still mentioning mongodb_consistent_backup is outdated by now too. I'll do it.

File: doc/source/release-notes/1.2.0.rst [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+inspired by (and intends to replace) the Percona-Lab/mongodb_consistent_backup tool.
```
Comment (akira-kurogane) [inline doc/source/release-notes/1.2.0.rst:14]:
```suggestion
across a MongoDB sharded cluster (or a single replica set).
```

**Pull request merged**